vispy/gloo/preprocessor.py

```diff
@@ -45,7 +45,7 @@ def replace(match):
                 logger.critical('"%s" not found' % filename)
                 raise RuntimeError("File not found", filename)
             text = '\n// --- start of "%s" ---\n' % filename
-            text += remove_comments(open(path).read())
+            text += open(path).read()
```

Comment:
Is that an OSX thing? I've never seen anything like that. At least on my system I can confirm using `os.fdopen` that descriptors are not leaked.